### ISSUE TITLE
Lasso/Marko production mode

### ISSUE THREAD

**Issue by juliocanares** [lasso-js/lasso#196]
Hi guys. I’ve been working a lot with Marko those days that I am now have so many pages using it.

I have a problem with that, since I am using [warmup](https://github.com/patrick-steele-idem/warmup) my deployment process has gotten too slow(it takes minutes to warmup) and it usually makes my process manager to run out of memory.

I would like to know if there is a way to make all that process before deployment.

another issue I am having is that I can’t hit any page using `warmup` that is blocked by credentials like the /onboard , /checkout, /account pages.

I already tried using lasso to generate the bundle and `cacheProfile` in the lasso config

```bash
lasso ./src/views/pages/home/browser.json --name home -c lasso-config.json --production
```

```javascript
require('lasso').configure({
  cacheProfile: 'production',
  bundleReadTimeout: 120000,
  bundlingEnabled: isProduction,
  plugins: [
    {
      plugin: 'lasso-less',
      config: {
        extensions: ['less'],
        lessConfig: {
          strictMath: true,
          strictUnits: true
        }
      }
    },
    'lasso-autoprefixer',
    'lasso-marko'
  ],
  outputDir: 'public',
  urlPrefix: '/public',
  minify: isProduction,
  fingerprintsEnabled: isProduction
});
```

**Comment by schetnikovich** [comment]
@juliocanares I'm also looking to some good solution for this problem. Please check third-party repository: [lasso-marko-extras](https://github.com/schetnikovich/lasso-marko-extras). It uses Lasso & Marko API to generate all static assets. Example of usage is in the [test/site](https://github.com/schetnikovich/lasso-marko-extras/tree/master/test/site) folder.

Key Lasso  and Marko APIs are:
1. `lasso.lassoResource()`
2. `lasso.lassoPage()`
3. `template.getDependencies()`

There is a similar question from my side [How to generate all static assets for the whole app?](#192)

**Comment by juliocanares** [comment]
@schetnikovich I will take a look :D
